packages/wrangler/src/reporting.ts

```diff
@@ -72,15 +82,17 @@ export async function reportError(err: Error, origin = "") {
       message: "Would you like to submit a report when an error occurs?",
       choices: [
         { title: "Always", value: true },
-        { title: "Yes", value: "once" },
+        { title: "Only this time", value: "once" },
         { title: "No", value: false },
       ],
       initial: 2,
     });
 
-    if (userInput.sentryDecision === "once") {
+    if (userInput.sentryDecision !== false) {
```

Comment:
this felt like a bug, it wouldn't track the current error if we chose "Always"

Reply:
I don't think this was a bug.
If you chose Always then the error gets logged below on line 96 (original).

Reply:
oh huh I missed that, you're right